docs/sdks/sdk-keys.md

```diff
@@ -12,13 +12,25 @@ From the feature flagging page, you can toggle which environment you are in by c
 
 When Eppo's SDK is initialized, only configurations from flags enabled in that environment will be downloaded. This can help when using flags across a variety of use cases and it's not necessary to download every flag for every instance of the Eppo client. A common example is when different flags are used in mobile and web applications. In this case, you can create separate Web and Mobile environments and use Eppo's UI to manage which flags are active in each.
 
+## Types of SDK keys
+There are 3 types of SDK keys - SDK keys itself, Client Tokens and Browser Extension keys. SDK keys and Client Tokens are used by SDKs. Browser Extension keys are used by [Eppo Developer Tools - Chrome Extension](/sdks/developer-tools-extension/). SDK keys and Client Tokens are the same functionality keys, except config always is obfuscated when Client Tokens are used. Therefore, the main application of the Client Tokens is SDK clients working in browsers. Meanwhile, the primary application of the SDK keys is on server side.
```

Comment:
SDK clients working in browsers. Matters for mobile too!

Perhaps "SDK clients requesting configurations for clients such as browsers or mobile applications."

Reply:
Updated here https://github.com/Eppo-exp/eppo-docs/pull/675/commits/2f5d7c42c5fe3fd21d915b74c5a4c00353f85921.